pkg/apis/core/v1beta1/defaults.go

```diff
@@ -164,7 +164,7 @@ func SetDefaults_Shoot(obj *Shoot) {
 			obj.Spec.Kubernetes.KubeControllerManager.NodeCIDRMaskSize = calculateDefaultNodeCIDRMaskSize(&obj.Spec)
 		}
 		if obj.Spec.Kubernetes.KubeControllerManager.NodeMonitorGracePeriod == nil {
-			obj.Spec.Kubernetes.KubeControllerManager.NodeMonitorGracePeriod = &metav1.Duration{Duration: 2 * time.Minute}
+			obj.Spec.Kubernetes.KubeControllerManager.NodeMonitorGracePeriod = &metav1.Duration{Duration: 40 * time.Second}
```

Comment:
For existing `Shoot`s, we also want to switch to `40s` when upgrading to `1.27`. However, this will not be sufficient since they will have set this field already to `2m`.
I think we should introduce code in `registry/core/shoot` and set this field to `40s` when the Kubernetes version is upgraded from `1.26` to `1.27` and when it was `2m` before. This way, we only set it to `40s` once when upgrading the Kubernetes version, but end users can change it later again if needed/desired.